src/data/crfParameters.ts

```diff
@@ -1,4 +1,12 @@
-export const CRF_ATTRIBUTES = [
+export type CrfParameter = {
+  officialName: string
+  customName: string
+  att_type: 'identifying' | 'quasiidentifying' | 'insensitive' | 'sensitive'
```

Comment:
what does "att" mean ?

Reply:
attribute I guess. I kept the naming already in place

Reply:
ok can you rename it to attr_type